HW02/Ilia_Aksenov/ilia_aksenov_hw_02_t_01.rb

```diff
@@ -0,0 +1,8 @@
+def parse(filepath)
+  data = File.open(filepath).readlines.map(&:chomp)
```

Comment:
We can use a short version 
```suggestion
  data = File.readlines(filepath, chomp: true)
```

Reply:
And I would suggest splitting this method into 2 ( `read_file` and `find_errors`)